src/shared/src/notification.ts

```diff
@@ -43,10 +43,10 @@ export class NotificationManager {
     }
 
     registerIntercepts() {
-        this.manager.interceptKey('PAUSE_TOGGLE', true);
-        this.manager.interceptKey('PAUSE_ON', true);
-        this.manager.interceptKey('PAUSE_OFF', true);
-        this.manager.interceptKey('PAUSE_SET', true);
+        this.manager.interceptKey('PAUSE_TOGGLE', false);
+        this.manager.interceptKey('PAUSE_ON', false);
+        this.manager.interceptKey('PAUSE_OFF', false);
+        this.manager.interceptKey('PAUSE_SET', false);
```

Comment:
```suggestion
        this.manager.interceptKey('PAUSE_TOGGLE', true);
        this.manager.interceptKey('PAUSE_ON', true);
        this.manager.interceptKey('PAUSE_OFF', true);
        this.manager.interceptKey('PAUSE_SET', true);
```
Second param means pass through to sim.